rfd/0239-windows-ca-split.md

```diff
@@ -0,0 +1,166 @@
+---
+authors: Alan Parra (alan.parra@goteleport.com)
+state: draft
+---
+
+# RFD 0239 - Windows CA split
+
+## Required Approvers
+
+* Engineering: @rosstimothy && @zmb3
+* Security: (@rob-picard-teleport || @klizhentas)
+* Product: @klizhentas
+
+## What
+
+Split the "windows" (aka Desktop Access) CA from the User CA.
```

Comment:
Should we refer to this as "windows-desktop" and the Windows Desktop CA? We will soon support Desktop Access for Linux, which won't use this CA and will still use the User CA. We may also have future Windows resources that may want/need to use their own CA.